src/backend/src/players/players.service.ts

```diff
@@ -0,0 +1,46 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Player } from '../entities/player.entity';
+import { Repository } from 'typeorm';
+import { Game } from '../entities/game.entity';
+
+@Injectable()
+export class PlayersService {
+  constructor(
+    @InjectRepository(Player)
+    private playersRepository: Repository<Player>,
+    @InjectRepository(Game)
+    private readonly gamesRepository: Repository<Game>,
+  ) {}
+
+  async findAll(): Promise<Player[]> {
+    return this.playersRepository.find();
+  }
+
+  async findOne(id: number): Promise<Player> {
+    const player = this.playersRepository.findOne(id);
+    if (!player) {
+      throw new BadRequestException('Player not found');
+    }
```

Comment:
are you sure you want to throw an exception here? As opposed to just returning undefined? I feel like when i call a findOne method I don't expect it to throw, but up to you